data/language/dutch.txt

```diff
@@ -1171,13 +1171,13 @@ STR_1169    :(Geen)
 STR_1170    :{STRING}
 STR_1171    :{RED}Gesloten - - 
 STR_1172    :{YELLLOW}{STRINGID} - - 
-STR_1173    :{SMALLFONT}{BLACK}Voetpaden en wachtrijen neerzetten
+STR_1173    :{SMALLFONT}{BLACK}Voetpaden en wachtrijen aanleggen
 STR_1174    :Banier in de weg
 STR_1175    :Kan dit niet op hellend voetpad neerzetten
-STR_1176    :Kan hier geen voetpad neerzetten...
+STR_1176    :Kan hier geen voetpad aanleggen...
 STR_1177    :Kan dit voetpad niet verwijderen...
 STR_1178    :Landhelling is ongeschikt
-STR_1179    :Voetpad staat in de weg
+STR_1179    :Voetpad ligt in de weg
```

Comment:
Ik begrijp waarom je dit aankaart. Maar stel dat je een achtbaan aan het bouwen bent en je komt tegen een voetpad/achtbaan aan dan ligt/staat het 'stil' in de weg. In de weg liggen/staan is zeg maar alleen A. Ik hoop dat jullie zo begrijpen wat ik bedoel.

Overigens ligt een 'hoog' voetpad nog steeds, maar dan op een constructie van palen. Denk aan dakpannen, die liggen ook op je dak.